pkg/server/container_create.go

```diff
@@ -213,7 +216,13 @@ func (c *criContainerdService) generateContainerSpec(id string, sandboxPid uint3
 
 		// TODO(random-liu): [P1] Set selinux options.
 
-		// TODO(random-liu): [P2] Add apparmor and seccomp.
+		// TODO(random-liu): [P2] Add seccomp.
+
+		// Set apparmor profile name, Note: trims default profile name prefix
+		appArmorProf := securityContext.GetApparmorProfile()
```

Comment:
Yeah was interested to see what docker/default and/or runtime/default means in this context. Are we expected to define it or kubelet/cri?  Wasn't sure just how much filtering we should do and how much to let pass through.  But yes at a minimum a TODO is needed here. I was exploring such defaults to a higher degree on the seccomp stuff.